sites/partners/src/listings/Aside.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useContext, useMemo, useCallback } from "react"
```

Comment:
I see you get rid of the `saveAndExit ` function, I think we can also remove `useCallback` and a couple of another unused elements in this file.